docs/FAQ.md

```diff
@@ -0,0 +1,10 @@
+# Frequently Asked Questions
+
+1. **How can I launch a cluster built from multiple PRs, on GCP?**
+   
+    `launch openshift/origin#49563,openshift/kubernetes#731,openshift/machine-api-operator#831 gcp`
+   
+
+2. **How can I update the "Global Cluster Pull Secret" on a cluster-bot created cluster?**
+
+    Follow the instructions [here](https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#images-update-global-pull-secret_opting-out-remote-health-reporting).
```

Comment:
Might be worth linking to https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/ to get a valid pull secret if you don't have one locally but lgtm.

Reply:
Done!